docs/Documentation.md

```diff
@@ -207,7 +207,8 @@ Gets the whole queue
 #### `removeUpcomingTracks()`
 Clears any upcoming tracks from the queue.
 
-#### `updateMetadata(id, metadata)`
+#### `updateMetadataForTrack(id, metadata)`
+Updates the metadata of a track in the queue.
```

Comment:
I would also add something like: `If the metadata of the current track is changed -- the Android Notification and the NowPlaying Center will be updated accordingly.